src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+[[I walked into the sign’s field of light|The Gutter - Outside hub]].]]]]<gutterSubSign|
+<!-- This makes the blinking neon sign effect work -->
+(set: $guttersign1 to "<span class='neon'><span class='blinking'>THE</span> GUTTER</span>")(set: $guttersign2 to "<span class='neon'>THE <span class='blinking2'>G</span>UTTER</span>")(set: $guttersign3 to "<span class='neon'>THE GUTT<span class='blinking3'>ER</span></span>")
+<style> .neon {
+  font-size: 2em;
+  font-family: copperplate;
+  color: transparent;
+	text-shadow:
+		0em    0em 0.02em rgba(255,255,255,0.75),
+		-0.2em 0em 0.5em  rgba(255,255,255,0.5),
+		0.2em  0em 0.5em  rgba(255,255,255,0.5);
+	text-decoration: none;
+}
+.blinking {
+  animation: fade-in-out .3s ease-in-out infinite alternate;
+	-webkit-animation: fade-in-out .3s ease-in-out infinite alternate;
+}
+.blinking2 {
+  animation: fade-in-out 1.2s ease-in-out infinite alternate;
+	-webkit-animation: fade-in-out 1.2s ease-in-out infinite alternate;
+}
+.blinking3 {
+  animation: fade-in-out .8s ease-in-out infinite alternate;
+	-webkit-animation: fade-in-out .8s ease-in-out infinite alternate;
+}
+</style>
+<!--These are the variables that need to be set for the Outside hub passages to work properly-->
+(set: $oldSchoolJuggalosSeen to false)(set: $jarSeen to false)(set: $justLeftTheGutter to false)(set: $golfCartSeen to false)(set: $cartPartsSeen to 0)(set: $wheelsSeen to false)(set: $engineSeen to false)(set: $defenseSeen to false)(set: $hornSeen to false)(set: $gutterStatus to "intro")(set: $gutterOutside to "hub")
+
+::The Gutter - Name prompt outside hub
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|The Gutter - Outside hub]].]
+}(else:)[No, [[that's not right|The Gutter - Name prompt outside hub]].]
+
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
+(set: $oldSchoolJuggalosSeen to false)(set: $jarSeen to false)(set: $justLeftTheGutter to false)(set: $golfCartSeen to false)(set: $cartPartsSeen to 0)(set: $wheelsSeen to false)(set: $engineSeen to false)(set: $defenseSeen to false)(set: $hornSeen to false)(set: $gutterStatus to "intro")
+
+::The Gutter - Outside hub
+(if: $justLeftTheGutter is true)[(set: $justLeftTheGutter to false)I walked back into the dim neon light of the Gutter’s sign. I could see:]
+\(else:)[In the dim light of the neon sign, I could see:]
+
+>>[[The Gutter’s beaten-to-hell block of a front door|The Gutter - Inside $gutterStatus]]
+>>[[The paved path to The Church|The Church - intro]]
+>>[[The dirt road to the Roach Market|RM-$roachEntrance-entrance]]
+(if: $oldSchoolJuggalosSeen is false)[>>[[Some old-school juggalos in a heated conversation by the side of the building|The Gutter - Outside - Old-school juggalos]]]
```

Comment:
If we want empty lines to appear where these options were, we can leave this as-is. If we want to collapse them, we can use the trick I used in the Tent City chase:
```
>>[[Previous option|Previous option]]\
(if: $oldSchoolJuggalosSeen is false)[
>>[[Some old-school juggalos in a heated conversation by the side of the building|The Gutter - Outside - Old-school juggalos]]]
```